### PR TITLE
docs: adds direct downloads section [sc-00]

### PR DESCRIPTION
**Pull request by tnolet**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Notes for the Reviewer
- adds section to CLI installation docs to directly download a CLI distribution

## Screenshots
image

### CODE REVIEW

**Review by checklyalex — APPROVED** [review]
LGTM

File: site/content/docs/cli/installation.md [hunk 1]
```diff
@@ -96,6 +96,25 @@ before hand or just sign up for a new account straight from the terminal.
 ```bash
 npx checkly login
 ```
+## Direct download
+
+If you cannot access the npm registry directly, you can also download the Checkly CLI via our CDN.
+
+- [MacOS / Darwin](https://cdn.checklyhq.com/downloads/checkly-cli/4.0.7/darwin/checkly.zip)
```
Comment (tnolet) [inline site/content/docs/cli/installation.md:103]:
```suggestion
- [MacOS / Darwin](https://cdn.checklyhq.com/downloads/checkly-cli/4.0.8/darwin/checkly.zip)
```

File: site/content/docs/cli/installation.md [hunk 2]
```diff
@@ -96,6 +96,25 @@ before hand or just sign up for a new account straight from the terminal.
...
+- [Windows](https://cdn.checklyhq.com/downloads/checkly-cli/4.0.7/windows/checkly.zip)
```
Comment (tnolet) [inline site/content/docs/cli/installation.md:104]:
```suggestion
- [Windows](https://cdn.checklyhq.com/downloads/checkly-cli/4.0.8/windows/checkly.zip)
```

**Review by tnolet — COMMENTED** [review]
change the versions

**Review by checklyalex — APPROVED** [review]
LGTM

**Pull request merged**